.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 env:
   global:
-    - STARFISH_VALIDATION_STRICT=true
+    - STARFISH_CONFIG=@.travis-config.json
```

Comment:
I'm concerned that this is a relative path.  I would prefer that we copy it to an absolute path, and reference an absolute path.